team-cost-reporter/plugins/dynamodb/__init__.py

```diff
@@ -82,6 +84,36 @@ def _get_table_info(account_creds, region_list, debug=False):
     return account_dynamo_details
 
 
+def _is_match(cc_env_name, config_match_env, debug=False):
```

Comment:
My only comment here is around variable names - could probably be more generic, like 'string_to_check' and ?? No idea...  :)